rails-js/controllers/index.js

```diff
@@ -1,4 +1,23 @@
-import InputMaskController from "./input_mask_controller";
+import { Application } from "@hotwired/stimulus";
+
+// Shared controllers
 import ToggleController from "./toggle_controller";
 
-export { InputMaskController, ToggleController };
+const registerStimulusControllers = (appDefinitions) => {
```

Comment:
What's going on in this method? Is this ported over from somewhere?

Reply:
Yep -- I'm trying to extract out some of the complication needed to register all the app's stimulus controllers (see `app/javascript/controllers/application.js` in OS or Buyout)... LMK if it's more confusing split up this way, though -- I'd _hoped_ to be able to extract the entire file away, but it turns out esbuild needs the `import { definitions } from "stimulus:./";` line to live in the consuming app, rather than shared, because that esbuild plugin relies on the path on disk to autoload all the app's stimulus controllers. :/